### DISCUSSION TITLE
GitHub Copilot error after successfully installing and setup in my VC Studio

### DISCUSSION THREAD

**Discussion by twt2shiv** [community/community#16427] [Copilot Conversations] [answered]
Hello to All,

I am not able to use GitHub Copilot after successfully installing and setup in my visual code studio.
The error I am getting is that '**GitHub Copilot could not connect to the server. Extension activation failed: "User not authorized"**' and in the Logs says "**[INFO] [auth] [2022-05-12T04:31:00.161Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-05-12T04:31:00.162Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized**""

Please help me, with how I activate it same.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
